### PR TITLE
PLAT-143745 Added screenshot tests for Sprite

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Sprite component had no screenshot tests.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
I added tests for Sprite component.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-143745

### CODE REVIEW

File: tests/screenshot/apps/components/Sprite.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import Sprite from '../../../../Sprite';
+import {withConfig} from './utils';
```
Comment (stanca-pop-lgp) [inline tests/screenshot/apps/components/Sprite.js:2]: Please add an empty row here.

Comment (alexandrumorariu) [reply tests/screenshot/apps/components/Sprite.js:3]: Done

File: tests/screenshot/apps/components/Sprite.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+];
+
+const SpriteTests = [
+	...LTR,
+
+	// RTL
+	...withConfig({locale: 'ar-SA'}, LTR),
+
+	// Tallglyphs
+	...withConfig({locale: 'vi-VN'}, LTR)
```
Comment (stanca-pop-lgp) [inline tests/screenshot/apps/components/Sprite.js:20]: I think tests with Tallglyphs are not relevant to this component.

Comment (alexandrumorariu) [reply tests/screenshot/apps/components/Sprite.js]: Done

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Comment by dongsuwon** [comment]
LGTM

**Pull request merged**